btr-api/src/btr_api/schemas/SignificantIndividualsFiling.json

```diff
@@ -0,0 +1,176 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "FilingActionE": {
+      "enum": ["add", "edit", "remove"],
```

Comment:
i don't think we should have actions like this.
in other filings, and it looks like bods is the same, that a submission is for the entire current state.
as such, if a person has an identifier they get updated, if they don't have one they're created and if they are in the longitudinal record but not in the filing they are removed.